pyproject.toml

```diff
@@ -20,6 +20,7 @@ dependencies = [
   "mpi4py; python_version < '3.13'",
   "h5py>3.12.1",
   "libsupermesh",
+  "loopy>2024.1",
```

Comment:
Should this not be 2025.1?

Reply:
It can either be `>2024.1` or `>=2025.1`, as they are equivalent.

Personally, I'd think that `>=2025.1` will be easier to maintain, because it will arguably be easier in future to answer the question "what was the version that added that feature I really needed" rather than "what was the latest version that didn't have that feature"...

Reply:
I guess. I was thinking in terms of "what versions are illegal". They are obviously equivalent so happy to change if there's another preference.

Reply:
ah. I don't mind.